src/components/Questions/Body.vue

```diff
@@ -319,18 +321,12 @@ export default defineComponent({
       state.isImageLoading = true
     }
 
-    function checkIfNumberContainsDecimal(x: Number | null) {
+    function doesNumberContainDecimal(x: Number | null) {
       return String(x).includes(".")
     }
 
-    function isNumberLarge(x: Number) {
-      const maxValue: number = 1e10
-      return x > maxValue
-    }
-
-    function checkIfNumberExceedsDecimalDigitLimit(x: Number | null) {
-      const maxDecimalPlaces: number = 10 // max length of digits after decimal
-      return String(x).includes(".") && String(x).split(".")[1].length > maxDecimalPlaces - 1
+    function doCharactersExceedLimit(x: Number | null) {
```

Comment:
this is specific for numerical answers no?
For subject questions, we have another limit that comes from the backend
I think it wouldn't create any confusion if the name of this method reflected that we're checking for limit of a numerical question and not any other question type

Reply:
Same case with the variable `MAX_LENGTH_CHARACTERS`.
The `characters` word might confuse someone between subjective questions and numerical questions.

Reply:
shall i name it doNumericalCharacters instead? and `MAX_LENGTH_NUMERICAL_CHARACTERS`

Reply:
Yes sounds good